src/kw/kw.cpp

```diff
@@ -8,16 +8,33 @@
  */
 
 #include <string>
+#include <ctime>
+
 using std::string;
 
 #include "mercury/kw/kw.h"
 
+#include <stdlib.h>
 #include <parson.h>
 #include <curl/curl.h>
 
 
+KanoWorld::KanoWorld(void)
+{
+    // Data file to store token and expiration date
+    data_filename = string(getenv("HOME")) + "/" + ".mercury_kw.json";
+    token = "";
+    expiration_date = "";
+}
+
+KanoWorld::~KanoWorld(void)
+{
+}
```

Comment:
Do we need the destructor explicitly defined if we aren't doing anything?

Reply:
True, not anymore in this case. removed